xml/System/Type.xml

```diff
@@ -13225,9 +13240,16 @@ Types that are defined in the .NET Standard are not marked with <xref:System.Ser
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value that indicates whether the <see cref="T:System.Type" /> is a type definition.</summary>
+        <value><see langword="true" /> if the current <see langword="Type" /> is a type definition; otherwise, <see langword="false" />.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the current <see langword="Type" /> is a type definition; otherwise, <see langword="false" />.</value>
```